tests/scripts/generate_tls13_compat_tests.py

```diff
@@ -71,18 +71,26 @@
     'x448': 0x1e,
 }
 
+
 class TLSProgram(metaclass=abc.ABCMeta):
     """
     Base class for generate server/client command.
     """
-
-    def __init__(self, ciphersuite, signature_algorithm, named_group, compat_mode=True):
+    # pylint: disable=too-many-arguments
+    def __init__(self, ciphersuite=None, signature_algorithm=None, named_group=None,
+                 cert_sig_alg=None, compat_mode=True):
         self._ciphers = []
         self._sig_algs = []
```

Comment:
Do we still need _sig_algs?

Reply:
Yes we just want introduce a new attribute _cert_sig_algs to define the certificate(s) to be used independently of the definition of the forced signature algorithms _sig_algs.